api/spec/v1.yaml

```diff
@@ -2056,6 +2056,33 @@ components:
         Values of EVM type `address` and `address payable` are represented as lowercase hex strings with a "0x" prefix.
         Values of EVM type `bytes` and `bytes<N>` are represented as base64 strings.
         Values of other EVM types (integer types, strings, arrays, etc.) are represented as their JSON counterpart.
+      
+    RuntimeEvmContract:
+      type: object
+      required: [creation_tx, creation_bytecode, verified]
+      properties:
+        creation_tx:
+          type: string
+          description: The Oasis cryptographic hash of the transaction that created the smart contract.
+        creation_bytecode:
```

Comment:
Just to have a discussion thread:
Reminder that you said you'd up the number of rounds in the e2e regression test. Maybe 10k is good? It takes almost exactly 2min on my machine, which feels like a decent amount of time to me. Or maybe 5k.

Might be worth a separate PR (before this one goes in) just because of all the diff noise. Your call.